geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureTypes.scala

```diff
@@ -24,7 +24,7 @@ import org.opengis.feature.simple.SimpleFeatureType
 import scala.collection.JavaConversions._
 import scala.util.parsing.combinator.JavaTokenParsers
 import scala.util.{Failure, Success, Try}
-
+import RichSimpleFeatureType._
```

Comment:
RichSimpleFeatureType holds the constants for the key and the delimiter, but I can fully qualify